src/lib/bridge/handlers/open_options_page.ts

```diff
@@ -0,0 +1,19 @@
+import {RequestHandler} from '../types';
+import {RequestType} from './types';
+import {isFirefox} from '../../utils/detect';
+
+class OpenOptionsPageHandler implements RequestHandler<undefined, void> {
+    getType(): RequestType {
+        return RequestType.OPEN_OPTIONS_PAGE;
+    }
+
+    async handleRequest(request: undefined, sender: chrome.runtime.MessageSender): Promise<void> {
+        if (isFirefox()) {
```

Comment:
Replace with `runtimeNamespace().runtime.openOptionsPage()`

Reply:
the one returned by firefox is a promise while chrome’s isnt